src/vsd/corona/import/import_handhaving.py

```diff
@@ -56,6 +62,24 @@ def main() -> None:
         header=0,
     )
     df.index.name = "id"
+
+    df = df.drop(['ois_week_nummer'], axis = 1)
+
+    # list of conditions
```

Comment:
Hopelijk is de corona volgend jaar voorbij :-)  Anders hebben we hier een soort millenium-bug.
Ik zie dat er een libje `isoweek` is om dit wat generieker op te lossen. Maar, ik weet niet of dit de moeite waard is.